mapswipe-dashboard/BUILD.md

```diff
@@ -0,0 +1,43 @@
+## Webpack
```

Comment:
what do these mean? Can you add a bit of explanation at the start of the file to clarify?

Reply:
We will remove this. This is just a list of dependencies used on the initial template.